inductiva/resources/resources.py

```diff
@@ -0,0 +1,21 @@
+"""Methods related to managing executers and resource pools."""
+from uuid import UUID
+
+import inductiva
+from inductiva import api
+from inductiva.client import ApiClient, ApiException
+from inductiva.client.apis.tags.executers_api import ExecutersApi
+
+
+def create_machine_group_id() -> UUID:
```

Comment:
this runs on the server side or client side? 
ID's should be created on the serverside and returned as part of calls.